packages/itwinui-react/src/core/ComboBox/ComboBoxMenu.tsx

```diff
@@ -93,10 +89,6 @@ export const ComboBoxMenu = React.forwardRef((props, forwardedRef) => {
                   // set as constant because we don't want it shifting when items are unmounted
                   maxInlineSize: 0,
 
-                  // max-height must be on the outermost element for virtual scroll
-                  maxBlockSize:
-                    'calc((var(--iui-component-height) - 1px) * 8.5)',
-                  overflowY: isOverflowOverlaySupported() ? 'overlay' : 'auto',
```

Comment:
since we are now using `iui-menu` as the outer element, we don't need to set these inline styles for virtualization

while writing this, i also realized that we can remove `iui-scroll` and make it the default behavior. this will fix #1301 as well. i will create a separate PR for it